package.json

```diff
@@ -33,7 +33,10 @@
     "Data",
     "Visualization"
   ],
-  "author": "Curran Kelleher",
+  "authors": [
+    "Curran Kelleher",
+    "Anooj Pai"
```

Comment:
Nice change! It would be best to make a separate PR so this can get in, independently of the NGrok stuff.